app/models/account.rb

```diff
@@ -167,6 +167,8 @@ def schedule_backend_sync_worker
   has_many :sso_authorizations, through: :users
   has_many :user_sessions, through: :users
 
+  validates :org_name, format: { with: /\A.*[a-zA-Z0-9]+.*\z/, message: :invalid_format }
```

Comment:
By the way, I've just noticed there is another validation for `:org_name` on line 265:
```
validates :org_name, presence: true, length: { maximum: 255 }
```

I think we need to either merge the two validations, or at least to put them together, because it's hard to see then the complete set of rules for this field.

Reply:
Yes may  be we can merge them
Let me check and update the PR